lab_1_classify_by_unigrams/main.py

```diff
@@ -11,6 +13,14 @@ def tokenize(text: str) -> list[str] | None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
+    if isinstance(text, str):
+        tokens = []
```

Comment:
Can you try to refactor the code using list comprehention?

Reply:
done